### ISSUE TITLE
[Bug]:

### ISSUE THREAD

**Issue by AlfaScarlate** [FreeTubeApp/FreeTube#7830]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

When  starting some videos in other languages, the original language /audio track is not applied and is replaced by the AI voiceover dub of the video.
There is no option to change audio tracks between original and "AI dub" as well. (the icon doesn't show up)

### Expected Behavior

The video should be played in the original audio track and, if the video has a AI dub applied to it, then i should be able to change between the original audio and the AI dub if required.

### Issue Labels

accessibility issue

### FreeTube Version

v 0.23.6 beta

### Operating System Version

windows 10 22h2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.23.5

### Additional Information

Some videos have the option to change languages, but some videos do- from channels that are know to use their OG language (ex: PT-BR) - do not have such option. Most of them are being translated to the "AI dub" in english and the tracks cannot be switched.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Duplicate of #6683
